cdist/conf/type/__acl/explorer/acl_is

```diff
@@ -18,9 +18,15 @@
 # along with cdist. If not, see <http://www.gnu.org/licenses/>.
 #
 
-if [ -e "/$__object_id" ]
+[ ! -e "/$__object_id" ] && exit 0
+
+if ! command -v getfacl > /dev/null
 then
-    getfacl "/$__object_id" 2>/dev/null \
-        | grep -E '^((default:)?(user|group):[^:]|(default:)?mask::)' \
-        || true
+    echo 'getfacl not available' >&2
+    exit 1
 fi
+
+getfacl -E "/$__object_id" 2>/dev/null \
+    | grep -E '^(default:)?(user|group|(mask|other):):[^:]' \
+    | sed -r 's/\s*#.+$//' \
```

Comment:
oups. there shouldn't be `-E` for setfacl, because freebsd don't know it. will fix. thanks.

open, net and freebsd's sed support `-r` (Same as -E for compatibility with GNU sed), but not Solaris's sed. I'll figure something out with `grep -Eo`.